### PR TITLE
add h10 pcb

### PR DESCRIPTION
**Pull request by hineybush**
## Description

Add h10 numpad PCB to QMK

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by hineybush** [comment]
forgot to update the readme, will do soon

File: keyboards/hineybush/h10/h10.h [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_ALL(    \
```
Comment (mechmerlin) [inline keyboards/hineybush/h10/h10.h:29]: ALL should be lower case

```suggestion
#define LAYOUT_all(    \
```

File: keyboards/hineybush/h10/h10.h [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+{ \
+    {k00, k01, k02, k03}, \
+    {k10, k11, k12, k13}, \
+    {k20, k21, k22, k23}, \
+    {k30, k31, k32, k33}, \
+    {k40, k41, k42, k43}, \
+    {k50, k51, k52, k53}  \
+}
+
+#define LAYOUT_2U_ALL(  \
```
Comment (mechmerlin) [inline keyboards/hineybush/h10/h10.h:46]:
```suggestion
#define LAYOUT_2u_all(  \
```

File: keyboards/hineybush/h10/h10.h [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+{ \
+    {k00, k01, k02, k03}, \
+    {k10, k11, k12, k13}, \
+    {k20, k21, k22, KC_NO}, \
+    {k30, k31, k32, k33}, \
+    {k40, k41, k42, KC_NO}, \
+    {KC_NO, k51, k52, k53}  \
+}
+
+#define LAYOUT_SPLIT_PLUS(  \
```
Comment (mechmerlin) [inline keyboards/hineybush/h10/h10.h:63]:
```suggestion
#define LAYOUT_split_plus(  \
```

File: keyboards/hineybush/h10/h10.h [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+{ \
+    {k00, k01, k02, k03}, \
+    {k10, k11, k12, k13}, \
+    {k20, k21, k22, k23}, \
+    {k30, k31, k32, k33}, \
+    {k40, k41, k42, KC_NO}, \
+    {KC_NO, k51, k52, k53}  \
+}
+
+#define LAYOUT_SPLIT_ZERO(  \
```
Comment (mechmerlin) [inline keyboards/hineybush/h10/h10.h:80]:
```suggestion
#define LAYOUT_split_zero(  \
```

File: keyboards/hineybush/h10/h10.h [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+{ \
+    {k00, k01, k02, k03}, \
+    {k10, k11, k12, k13}, \
+    {k20, k21, k22, KC_NO}, \
+    {k30, k31, k32, k33}, \
+    {k40, k41, k42, KC_NO}, \
+    {k50, k51, k52, k53}  \
+}
+
+#define LAYOUT_SPLIT_ENTER(  \
```
Comment (mechmerlin) [inline keyboards/hineybush/h10/h10.h:97]:
```suggestion
#define LAYOUT_split_enter(  \
```

File: keyboards/hineybush/h10/h10.h [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+{ \
+    {k00, k01, k02, k03}, \
+    {k10, k11, k12, k13}, \
+    {k20, k21, k22, KC_NO}, \
+    {k30, k31, k32, k33}, \
+    {k40, k41, k42, k43}, \
+    {KC_NO, k51, k52, k53}  \
+}
+
+#define LAYOUT_SPLIT_ENTER_PLUS(  \
```
Comment (mechmerlin) [inline keyboards/hineybush/h10/h10.h:114]:
```suggestion
#define LAYOUT_split_enter_plus(  \
```

File: keyboards/hineybush/h10/h10.h [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+{ \
+    {k00, k01, k02, k03}, \
+    {k10, k11, k12, k13}, \
+    {k20, k21, k22, k23}, \
+    {k30, k31, k32, k33}, \
+    {k40, k41, k42, k43}, \
+    {KC_NO, k51, k52, k53}  \
+}
+
+#define LAYOUT_SPLIT_ZERO_PLUS(  \
```
Comment (mechmerlin) [inline keyboards/hineybush/h10/h10.h:131]:
```suggestion
#define LAYOUT_split_zero_plus(  \
```

File: keyboards/hineybush/h10/info.json [hunk 8]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "h10",
+    "url": "",
+    "maintainer": "hineybush",
+    "width": 5.25,
+    "height": 7.5,
+    "layouts": {
+        "LAYOUT_ALL": {
```
Comment (mechmerlin) [inline keyboards/hineybush/h10/info.json:8]:
```suggestion
        "LAYOUT_all": {
```

File: keyboards/hineybush/h10/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [0] = LAYOUT_2U_ALL(
```
Comment (mechmerlin) [inline keyboards/hineybush/h10/keymaps/default/keymap.c:21]:
```suggestion
    [0] = LAYOUT_2u_all(
```

File: keyboards/hineybush/h10/keymaps/default/keymap.c [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+        KC_ESC,   KC_PEQL,   KC_BSPC,  MO(1),
+        KC_NLCK,  KC_PSLS,   KC_PAST,  KC_PMNS,
+        KC_P7,    KC_P8,     KC_P9,
+        KC_P4,    KC_P5,     KC_P6,    KC_PPLS,
+        KC_P1,    KC_P2,     KC_P3,
+                  KC_P0,     KC_PDOT,  KC_PENT
+    ),
+    [1] = LAYOUT_2U_ALL(
```
Comment (mechmerlin) [inline keyboards/hineybush/h10/keymaps/default/keymap.c:29]:
```suggestion
    [1] = LAYOUT_2u_all(
```

File: keyboards/hineybush/h10/keymaps/via/keymap.c [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [0] = LAYOUT_ALL(
```
Comment (mechmerlin) [inline keyboards/hineybush/h10/keymaps/via/keymap.c:21]:
```suggestion
    [0] = LAYOUT_all(
```

File: keyboards/hineybush/h10/keymaps/via/keymap.c [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+        KC_ESC,   KC_PEQL,   KC_BSPC,  MO(1),
+        KC_NLCK,  KC_PSLS,   KC_PAST,  KC_PMNS,
+        KC_P7,    KC_P8,     KC_P9,    KC_TRNS,
+        KC_P4,    KC_P5,     KC_P6,    KC_PPLS,
+        KC_P1,    KC_P2,     KC_P3,    KC_TRNS,
+        KC_TRNS,  KC_P0,     KC_PDOT,  KC_PENT
+    ),
+    [1] = LAYOUT_ALL(
```
Comment (mechmerlin) [inline keyboards/hineybush/h10/keymaps/via/keymap.c:29]:
```suggestion
    [1] = LAYOUT_all(
```

File: keyboards/hineybush/h10/keymaps/via/keymap.c [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+        KC_VOLU,   KC_MPLY,   KC_MNXT,   KC_TRNS,
+        KC_VOLD,   KC_TRNS,   KC_TRNS,   KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS,   RESET
+    ),
+    [2] = LAYOUT_ALL(
```
Comment (mechmerlin) [inline keyboards/hineybush/h10/keymaps/via/keymap.c:37]:
```suggestion
    [2] = LAYOUT_all(
```

File: keyboards/hineybush/h10/keymaps/via/keymap.c [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+        KC_VOLU,   KC_MPLY,   KC_MNXT,   KC_TRNS,
+        KC_VOLD,   KC_TRNS,   KC_TRNS,   KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS
+    ),
+    [3] = LAYOUT_ALL(
```
Comment (mechmerlin) [inline keyboards/hineybush/h10/keymaps/via/keymap.c:45]:
```suggestion
    [3] = LAYOUT_all(
```

**Comment by mechmerlin** [comment]
I noticed that not all the LAYOUT macros are supported in the QMK Configurator. Is this intentional?

**Comment by zvecr** [comment]
We should also align some of those layout names to leverage community support, `numpad_6x4`/`ortho_6x4`.

File: keyboards/hineybush/h10/keymaps/default/config.h [hunk 15]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2020 hineybush
```
Comment (fauxpark) [inline keyboards/hineybush/h10/keymaps/default/config.h:1]: This file can be deleted.

File: keyboards/hineybush/h10/keymaps/default/keymap.c [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+void matrix_scan_user(void) {
+
+}
+
+bool led_update_user(led_t led_state) {
+    return true;
+}
+*/
```
Comment (fauxpark) [inline keyboards/hineybush/h10/keymaps/default/keymap.c:56]:
```suggestion
```

File: keyboards/hineybush/h10/keymaps/via/config.h [hunk 17]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2020 hineybush
```
Comment (fauxpark) [inline keyboards/hineybush/h10/keymaps/via/config.h:1]: As above.

File: keyboards/hineybush/h10/keymaps/via/keymap.c [hunk 18]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+void matrix_scan_user(void) {
+
+}
+
+bool led_update_user(led_t led_state) {
+    return true;
+}
+*/
```
Comment (fauxpark) [inline keyboards/hineybush/h10/keymaps/via/keymap.c:72]:
```suggestion
```

File: keyboards/hineybush/h10/readme.md [hunk 19]
```diff
@@ -0,0 +1,15 @@
+# h10
+
+![h10](https://imgur.com/a/H01u2Sw)
```
Comment (fauxpark) [inline keyboards/hineybush/h10/readme.md:3]:
```suggestion
![h10](https://i.imgur.com/PeMuPZir.jpg)
```

File: keyboards/hineybush/h10/h10.h (hunk 1 above)
Comment (drashna) [reply keyboards/hineybush/h10/h10.h:29]: Wouldn't this be better as: 
```suggestion
#define LAYOUT_ortho_6x4(    \
```

File: keyboards/hineybush/h10/h10.h [hunk 20]
```diff
@@ -0,0 +1,146 @@
...
+         k51, k52, k53  \
+) \
+{ \
+    {k00, k01, k02, k03}, \
+    {k10, k11, k12, k13}, \
+    {k20, k21, k22, KC_NO}, \
+    {k30, k31, k32, k33}, \
+    {k40, k41, k42, KC_NO}, \
+    {KC_NO, k51, k52, k53}  \
+}
```
Comment (drashna) [inline keyboards/hineybush/h10/h10.h:61]:
```suggestion
#define LAYOUT_numpad_6x4(  \
    k00, k01, k02, k03, \
    k10, k11, k12, k13, \
    k20, k21, k22,      \
    k30, k31, k32, k33, \
    k40, k41, k42,      \
    k51,      k52, k53  \
) \
{ \
    {k00, k01, k02, k03}, \
    {k10, k11, k12, k13}, \
    {k20, k21, k22, KC_NO}, \
    {k30, k31, k32, k33}, \
    {k40, k41, k42, KC_NO}, \
    {KC_NO, k51, k52, k53}  \
}
```

File: keyboards/hineybush/h10/info.json (hunk 8 above)
Comment (drashna) [reply keyboards/hineybush/h10/info.json:8]:
```suggestion
        "LAYOUT_ortho_6x4": {
```

File: keyboards/hineybush/h10/keymaps/default/keymap.c (hunk 9 above)
Comment (drashna) [reply keyboards/hineybush/h10/keymaps/default/keymap.c:21]:
```suggestion
    [0] = LAYOUT_ortho_6x4(
```

File: keyboards/hineybush/h10/keymaps/default/keymap.c (hunk 10 above)
Comment (drashna) [reply keyboards/hineybush/h10/keymaps/default/keymap.c:29]:
```suggestion
    [1] = LAYOUT_ortho_6x4(
```

File: keyboards/hineybush/h10/keymaps/via/keymap.c (hunk 11 above)
Comment (drashna) [reply keyboards/hineybush/h10/keymaps/via/keymap.c:21]:
```suggestion
    [0] = LAYOUT_ortho_6x4(
```

File: keyboards/hineybush/h10/keymaps/via/keymap.c (hunk 12 above)
Comment (drashna) [reply keyboards/hineybush/h10/keymaps/via/keymap.c:29]:
```suggestion
    [1] = LAYOUT_ortho_6x4(
```

File: keyboards/hineybush/h10/keymaps/via/keymap.c (hunk 13 above)
Comment (drashna) [reply keyboards/hineybush/h10/keymaps/via/keymap.c:37]:
```suggestion
    [2] = LAYOUT_ortho_6x4(
```

File: keyboards/hineybush/h10/keymaps/via/keymap.c (hunk 14 above)
Comment (drashna) [reply keyboards/hineybush/h10/keymaps/via/keymap.c:45]:
```suggestion
    [3] = LAYOUT_ortho_6x4(
```

File: keyboards/hineybush/h10/rules.mk [hunk 21]
```diff
@@ -0,0 +1,32 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
```
Comment (drashna) [inline keyboards/hineybush/h10/rules.mk:32]:
```suggestion
HD44780_ENABLE = no         # Enable support for HD44780 based LCDs

LAYOUTS = ortho_6x4 numpad_6x4
```

**Comment by hineybush** [comment]
> I noticed that not all the LAYOUT macros are supported in the QMK Configurator. Is this intentional?

yes.

File: keyboards/hineybush/h10/keymaps/default/readme.md [hunk 22]
```diff
@@ -0,0 +1,2 @@
+/* Copyright 2020 hineybush
```
Comment (fauxpark) [inline keyboards/hineybush/h10/keymaps/default/readme.md:1]:
```suggestion
```

File: keyboards/hineybush/h10/keymaps/via/readme.md [hunk 23]
```diff
@@ -0,0 +1,2 @@
+/* Copyright 2020 hineybush
```
Comment (fauxpark) [inline keyboards/hineybush/h10/keymaps/via/readme.md:1]:
```suggestion
```

File: keyboards/hineybush/h10/rules.mk [hunk 24]
```diff
@@ -0,0 +1,34 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/hineybush/h10/rules.mk:11]:
```suggestion
```

File: keyboards/hineybush/h10/rules.mk [hunk 25]
```diff
@@ -0,0 +1,34 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
```
Comment (fauxpark) [inline keyboards/hineybush/h10/rules.mk:32]:
```suggestion
```

File: keyboards/hineybush/h10/info.json [hunk 26]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "h10",
+    "url": "",
+    "maintainer": "hineybush",
+    "width": 5.25,
+    "height": 7.5,
+    "layouts": {
+        "LAYOUT_split_6x4": {
+            "layout": [{"label":"0,0", "x":0, "y":0}, {"label":"0,1", "x":1, "y":0}, {"label":"0,2", "x":2, "y":0}, {"label":"0,3", "x":3, "y":0}, {"label":"1,0", "x":0, "y":1.25}, {"label":"1,1", "x":1, "y":1.25}, {"label":"1,2", "x":2, "y":1.25}, {"label":"1,3", "x":3, "y":1.25}, {"label":"2,0", "x":0, "y":2.25}, {"label":"2,1", "x":1, "y":2.25}, {"label":"2,2", "x":2, "y":2.25}, {"label":"3,3", "x":3, "y":2.25, "h":2}, {"label":"2,3", "x":4.25, "y":2.25}, {"label":"3,0", "x":0, "y":3.25}, {"label":"3,1", "x":1, "y":3.25}, {"label":"3,2", "x":2, "y":3.25}, {"label":"3,3", "x":4.25, "y":3.25}, {"label":"4,0", "x":0, "y":4.25}, {"label":"4,1", "x":1, "y":4.25}, {"label":"4,2", "x":2, "y":4.25}, {"label":"5,3", "x":3, "y":4.25, "h":2}, {"label":"4,3", "x":4.25, "y":4.25}, {"label":"5,1", "x":0, "y":5.25, "w":2}, {"label":"5,2", "x":2, "y":5.25}, {"label":"5,3", "x":4.25, "y":5.25}, {"label":"5,0", "x":0, "y":6.5}, {"label":"5,1", "x":1, "y":6.5}]
```
Comment (noroadsleft) [inline keyboards/hineybush/h10/info.json:9]:
```suggestion
        "LAYOUT_ortho_6x4": {
            "layout": [{"label":"0,0", "x":0, "y":0}, {"label":"0,1", "x":1, "y":0}, {"label":"0,2", "x":2, "y":0}, {"label":"0,3", "x":3, "y":0}, {"label":"1,0", "x":0, "y":1.25}, {"label":"1,1", "x":1, "y":1.25}, {"label":"1,2", "x":2, "y":1.25}, {"label":"1,3", "x":3, "y":1.25}, {"label":"2,0", "x":0, "y":2.25}, {"label":"2,1", "x":1, "y":2.25}, {"label":"2,2", "x":2, "y":2.25}, {"label":"2,3", "x":3, "y":2.25}, {"label":"3,0", "x":0, "y":3.25}, {"label":"3,1", "x":1, "y":3.25}, {"label":"3,2", "x":2, "y":3.25}, {"label":"3,3", "x":3, "y":3.25}, {"label":"4,0", "x":0, "y":4.25}, {"label":"4,1", "x":1, "y":4.25}, {"label":"4,2", "x":2, "y":4.25}, {"label":"4,3", "x":3, "y":4.25}, {"label":"5,0", "x":0, "y":5.25}, {"label":"5,1", "x":1, "y":5.25}, {"label":"5,2", "x":2, "y":5.25}, {"label":"5,3", "x":4.25, "y":5.25}]
```

File: keyboards/hineybush/h10/info.json [hunk 27]
```diff
@@ -5,8 +5,8 @@
...
+        "LAYOUT_ortho_6x4": {
+            "layout": [{"label":"0,0", "x":0, "y":0}, {"label":"0,1", "x":1, "y":0}, {"label":"0,2", "x":2, "y":0}, {"label":"0,3", "x":3, "y":0}, {"label":"1,0", "x":0, "y":1.25}, {"label":"1,1", "x":1, "y":1.25}, {"label":"1,2", "x":2, "y":1.25}, {"label":"1,3", "x":3, "y":1.25}, {"label":"2,0", "x":0, "y":2.25}, {"label":"2,1", "x":1, "y":2.25}, {"label":"2,2", "x":2, "y":2.25}, {"label":"2,3", "x":3, "y":2.25}, {"label":"3,0", "x":0, "y":3.25}, {"label":"3,1", "x":1, "y":3.25}, {"label":"3,2", "x":2, "y":3.25}, {"label":"3,3", "x":3, "y":3.25}, {"label":"4,0", "x":0, "y":4.25}, {"label":"4,1", "x":1, "y":4.25}, {"label":"4,2", "x":2, "y":4.25}, {"label":"4,3", "x":3, "y":4.25}, {"label":"5,0", "x":0, "y":5.25}, {"label":"5,1", "x":1, "y":5.25}, {"label":"5,2", "x":2, "y":5.25}, {"label":"5,3", "x":4.25, "y":5.25}]
```
Comment (noroadsleft) [inline keyboards/hineybush/h10/info.json:9]: Slight mistake by me on the previous suggestion.

```suggestion
            "layout": [{"label":"0,0", "x":0, "y":0}, {"label":"0,1", "x":1, "y":0}, {"label":"0,2", "x":2, "y":0}, {"label":"0,3", "x":3, "y":0}, {"label":"1,0", "x":0, "y":1.25}, {"label":"1,1", "x":1, "y":1.25}, {"label":"1,2", "x":2, "y":1.25}, {"label":"1,3", "x":3, "y":1.25}, {"label":"2,0", "x":0, "y":2.25}, {"label":"2,1", "x":1, "y":2.25}, {"label":"2,2", "x":2, "y":2.25}, {"label":"2,3", "x":3, "y":2.25}, {"label":"3,0", "x":0, "y":3.25}, {"label":"3,1", "x":1, "y":3.25}, {"label":"3,2", "x":2, "y":3.25}, {"label":"3,3", "x":3, "y":3.25}, {"label":"4,0", "x":0, "y":4.25}, {"label":"4,1", "x":1, "y":4.25}, {"label":"4,2", "x":2, "y":4.25}, {"label":"4,3", "x":3, "y":4.25}, {"label":"5,0", "x":0, "y":5.25}, {"label":"5,1", "x":1, "y":5.25}, {"label":"5,2", "x":2, "y":5.25}, {"label":"5,3", "x":3, "y":5.25}]
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**